src/extension.ts

```diff
@@ -49,8 +49,9 @@ async function getClusterKubeconfig(target: AksClusterTreeItem): Promise<string
     }
     const client = new azcs.ContainerServiceClient(target.root.credentials, target.root.subscriptionId);  // TODO: safely
     try {
-        const accessProfile = await client.managedClusters.getAccessProfile(resourceGroupName, name, 'clusterUser');
-        const kubeconfig = accessProfile.kubeConfig!.toString();  // TODO: safely
+        const clusterUserCredentials = await client.managedClusters.listClusterUserCredentials(resourceGroupName, name);
+        const kubeconfigList = clusterUserCredentials.kubeconfigs!.filter((kubeInfo) => kubeInfo.name === "clusterUser");  // TODO: safely
+        const kubeconfig = kubeconfigList[0].value?.toString();
```

Comment:
`value` is nullable and not typed as string?  Fie!

Reply:
:) 
<img width="510" alt="Screen Shot 2020-06-21 at 7 18 07 PM" src="https://user-images.githubusercontent.com/6233295/85219000-0675d880-b3f4-11ea-8c88-0ab66b01e6aa.png">